File: python/operators/operators_logical.py
# Copyright (C) 2019-2026 Katarzyna Krasińska
# Languagium - https://github.com/katheroine/languagium
# Licensed under GPL-3.0 - see LICENSE.md
# SPDX-License-Identifier: GPL-3.0-or-later

a = True; b = False

c = a and b
print(f"{a} and {b} = {c}")
c = a and a
print(f"{a} and {a} = {c}")
c = b and b
print(f"{b} and {b} = {c}")
c = a or b
print(f"{a} or {b} = {c}")
c = a or a
print(f"{a} or {a} = {c}")
c = b or b;
print(f"{b} or {b} = {c}")

c = not a
print(f"not {a} = {c}")
c = not b
print(f"not {b} = {c}")

File: javascript/operators/operators_logical.js
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

let a = true, b = false;

c = a && b;
console.log(`${a} && ${b} = ${c}`);
c = a && a;
console.log(`${a} && ${a} = ${c}`);
c = b && b;
console.log(`${b} && ${b} = ${c}`);
c = a || b;
console.log(`${a} || ${b} = ${c}`);
c = a || a;
console.log(`${a} || ${a} = ${c}`);
c = b || b;
console.log(`${b} || ${b} = ${c}`);

c = !a;
console.log(`!${a} = ${c}`);
c = !b;
console.log(`!${b} = ${c}`);
